### PR TITLE
remove termbox keybindings and events handlers

### PR DESCRIPTION
**Pull request by lyricnz**
Probably missing some key bindings, or handers for alternate names

### CODE REVIEW

**Comment by lyricnz** [comment]
Still has some debug code, and the silly color mode. I'm sure I've broken some of the more obscure key-bindings also. 
Only ~150 lines left in compat.go including a couple of ugly globals.

**Comment by lyricnz** [comment]
The last commit breaks background color off the right-hand side of the screen. Investigating...

**Review by vuon9 — APPROVED** [review]

**Comment by miguelmota** [comment]
LGTM

File: cointop/keybindings.go [hunk 1]
```diff
@@ -2,173 +2,101 @@ package cointop
...
+		key = "Down"
+	case "arrowleft", "leftarrow":
+		key = "Left"
+	case "arrowright", "rightarrow":
+		key = "Right"
+	case "return":
+		key = "Enter"
+	case "escape":
+		key = "Esc"
+	case "pageup":
```
Comment (miguelmota) [inline cointop/keybindings.go:38]: this is this missing the `"pgup"` alias name

**Comment by miguelmota** [comment]
Is it possible to add a mouse click event to open up the selected coin row item in the browser? I think before it was mapped to the right mouse click (ideally it's a double click but not sure if it's possible)

**Comment by miguelmota** [comment]
I think we're missing some nil checks in the updated view code. Got a random panic

DeepinScreenshot_select-area_20211106181307

File: cointop/keybindings.go (hunk 1 above)
Comment (lyricnz) [reply cointop/keybindings.go:38]: This is only for keys that weren't handled natively by tcell (call it "backward compatability").  Tcell has
```
	KeyPgDn:           "PgDn",
	KeyPgUp:           "PgUp",
```

**Comment by lyricnz** [comment]
> Is it possible to add a mouse click event to open up the selected coin row item in the browser? I think before it was mapped to the right mouse click (ideally it's a double click but not sure if it's possible)

I never heard of this feature before.  At least on my terminal, right-mouse is captured by the GUI not the terminal.  Feature request?

**Comment by lyricnz** [comment]
> I think we're missing some nil checks in the updated view code. Got a random panic
> 
> DeepinScreenshot_select-area_20211106181307

Is that on this branch? That stack trace doesn't align with code in tcell-events branch...?

**Comment by miguelmota** [comment]
> Is that on this branch? That stack trace doesn't align with code in tcell-events branch...?

@lyricnz my mistake; I had merged the other PR into this branch, so the issue might be from https://github.com/lyricnz/cointop/pull/4/files 

Moving issue comment over

**Comment by lyricnz** [comment]
> Moving issue comment over

I've just merged that into here, so problem may have come back here.  Is it possible that views don't have backing?  (if so, what to do with call?)

```
// SetStyle sets the selection text style for the view
func (view *View) SetSelStyle(st tcell.Style) {
	view.backing.SelStyle = st
}
```

**Comment by miguelmota** [comment]
> I never heard of this feature before. At least on my terminal, right-mouse is captured by the GUI not the terminal. Feature request? 

@lyricnz interesting, must depend on the terminal. The feature is currently there on master (works for me on alacritty)

https://github.com/cointop-sh/cointop/blob/9a9ee307d49de6f0e7d9f28a8db3afd7dbc37ea9/cointop/navigation.go#L579

**Comment by miguelmota** [comment]
> I've just merged that into here, so problem may have come back here. Is it possible that views don't have backing? (if so, what to do with call?)

we should check if it has a backing before modifying property ([example](https://github.com/cointop-sh/cointop/blob/9a9ee307d49de6f0e7d9f28a8db3afd7dbc37ea9/pkg/ui/view.go#L218))

File: cointop/keybindings.go (hunk 1 above)
Comment (miguelmota) [reply cointop/keybindings.go:38]: ah got it

**Pull request merged**